plugins/ote-migration/README.md

```
@@ -0,0 +1,454 @@
+
+The `tests-ext-copy` target is designed for CI pipelines that collect build artifacts from the `_output/` directory.
+
+### Test Lifecycle (Informing by Default)
+
+All migrated tests are automatically set to **Informing** lifecycle - they will run but won't block CI on failure.
+
+```go
+// Set lifecycle for all migrated tests to Informing
+// Tests will run but won't block CI on failure
+specs.Walk(func(spec *et.ExtensionTestSpec) {
+	spec.Lifecycle = et.LifecycleInforming
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)